dashboard/app/models/course.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: courses
+#
+#  id            :integer          not null, primary key
+#  name          :string(255)
+#  properties    :text(65535)
+#  plc_course_id :integer
```

Comment:
Thinking a little more - why not remove plc_course_id and add course_id to plc_courses?